lib/util/template/index.ts

```diff
@@ -87,6 +88,31 @@ handlebars.registerHelper({
   },
 });
 
+handlebars.registerHelper('lookupArray', (obj, key, options) => {
+  return (
```

Comment:
```suggestion
    if (
      !obj ||
      !is.nonEmptyStringAndNotWhitespace(key) ||
      !options ||
      !('lookupProperty' in options)
    ) {
      return [];
    }

  return (
```

validation

Reply:
Handlebars' own `lookup` does none of this: https://github.com/handlebars-lang/handlebars.js/blob/master/lib/handlebars/helpers/lookup.js

https://handlebarsjs.com/api-reference/helpers.html `lookupProperty` isn't described as "optional" - as far as I understand, it's always passed and is part of the API.

Null key doesn't cause any error. Handlebars' `lookup` helper simply passes key to `lookupProperty` as is.

If obj is null, it'll be converted to `[null]` and then filtered out by `!handlebars.Utils.isEmpty()`.

And if I'm passing wrong number of arguments, I'd prefer an error, not just silent empty result.

Reply:
add tests

Reply:
I already did it: https://github.com/renovatebot/renovate/pull/30618/commits/5dba00c1629abcb546b2ef9079e5d6412bdf3a87

is anything missing?